### PR TITLE
Nature40

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

File: src/app/operators/dialogs/nature40-catalog/nature40-catalog.component.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<wave-sidenav-header>Nature 4.0 Catalog</wave-sidenav-header>
+<ng-container [ngSwitch]="(catalog$ | async)?.size">
+    <div *ngSwitchCase="undefined" class="error">
+        <mat-spinner></mat-spinner>
+    </div>
+    <div *ngSwitchCase="0" class="error mat-typography"><p>The catalog is currently empty.</p></div>
```
Comment (jdroenner) [inline src/app/operators/dialogs/nature40-catalog/nature40-catalog.component.html]: is the error class the correct one for loading / empty catalogs?

File: src/app/app.component.ts [hunk 2]
```diff
@@ -352,7 +392,7 @@ export class AppComponent implements OnInit, AfterViewInit {
         );
     }
 
-    private setTheme(project: 'EUMETSAT' | 'GFBio' | 'GeoBon') {
+    private setTheme(project: 'EUMETSAT' | 'GFBio' | 'GeoBon' | 'Nature40') {
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply src/app/app.component.ts]: 9fcd61c

File: src/app/users/user.service.ts [hunk 3]
```diff
@@ -226,6 +226,104 @@ export class UserService {
         });
     }
 
+    /**
+     * Login using a Java Web Token. If it was successful, set a new user.
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply src/app/users/user.service.ts]: 735a77f

File: src/app/operators/dialogs/nature40-catalog/nature40-catalog.component.html (hunk 1 above)
Comment (ChristianBeilschmidt) [reply src/app/operators/dialogs/nature40-catalog/nature40-catalog.component.html]: 164cfcf

**Pull request merged**